build.gradle

```diff
@@ -45,6 +45,18 @@ project.ext {
   buildVersionFileName = "cruise-control-version.properties"
   commitId = project.hasProperty('commitId') ? commitId : null
   scalaBinaryVersion = getScalaBinaryVersion(scalaVersion)
+
+  defaultTestJvmArgs = []
+  // "JEP 403: Strongly Encapsulate JDK Internals" causes some tests to fail when they try
+  // to access internals (often via mocking libraries). We use `--add-opens` as a workaround
```

Comment:
often or always? As I see it is only used in the test task def, so perhaps we could rephrase this.

Reply:
I think it's more about the occurrences rather than temporal frequency. OFC those tests always fail without the add-opens.
BTW I copied it from Kafka :D. Most of the cases tests were failing because of easymock and powermock to be specific, but I have no clue why exactly `java.time` had to be opened, gradle wasn't kind enough to provide me more info about the failing test(s) before I added it.

Reply:
In Kafka easymock/powermock was replaced with mockito over time. It's still not done, but that's the way to get rid of these add-opens lateron.

Reply:
Yea I agree that we should rather use Mockito, however I think that comes with some test refactoring as well, so let's not do that now.